assets/airports/kric.json

```diff
@@ -0,0 +1,9359 @@
+            "name": ["2", "20"],
+            "end": [
+                ["N37d29.932348", "W77d19.779457", "158ft"],
+                ["N37d30.991123", "W77d19.461292", "165ft"]
+            ],
+            "ils": [true, false]
+        },	
+        {
+            "name": ["16", "34"],
+            "end": [
+                ["N37d30.998187", "W77d19.414595", "166ft"],
+                ["N37d29.747552", "W77d18.413082", "160ft"]
+            ],
+            "ils": [true, true]
+        }
+    ],
+    "airways": {
+        "J191": ["HPW", "PELTO", "ILM"],
+        "V1": ["SBY", "MAGGO", "JAMIE"],
+        "V20": ["OTT", "DAILY", "JUDED", "WHINO", "COLIN", "DUPEY", "TAPPA", "SVILL", "RIC"],
+        "V38": ["GVE", "EMMIE", "ROOKY", "RIC", "NEMAN", "HCM", "JUDLU", "TERKS", "CCV"],
+        "V155": ["RDU", "WIPER", "BEXGO", "LVL"],
+        "V157": ["LVL", "MANGE", "DALTO", "MATOX", "RIC", "SVILL", "TAPPA", "DUPEY", "COLIN", "PXT"],
+        "V260": ["FKN", "WAIKS", "HPW", "RIC", "FAK"]
+    },
+    "sids": {
+        "COLIN8": {
+            "icao": "COLIN8",
+            "name": "Colin Eight",
+            "rwy": {
+                "KRIC2": [["^_RW021DME", "A007+"], "_RIC05010"],
+                "KRIC16": [["^_RW161DME", "A007+"], "_RIC12107", "_RIC07410", "MODEL"],
+                "KRIC20": [["^_RW201DME", "A007+"], "_RIC12107", "_RIC07410", "MODEL"],
+                "KRIC34": [["^_RW341DME", "A007+"], "_RIC05010"]
+            },
+            "body": ["COLIN"],
+            "exitPoints": {
+                "COLIN": []
+            },
+            "draw": [
+                ["_RW021DME", "_RIC05010", "_RW341DME"],
+                ["_RIC05010", "COLIN*"],
+                ["_RW161DME", "_RIC12107", "_RW201DME"],
+                ["_RIC12107", "_RIC07410", "MODEL", "COLIN"]
+            ]
+        },
+        "KALLI7": {
+            "icao": "KALLI7",
+            "name": "Kalli Seven",
+            "rwy": {
+                "KRIC2": [["^_RW021DME", "A007+"], ["SUGGR", "A50+"]],
+                "KRIC16": [["^_RW161DME", "A007+"], ["ITTEM", "A50+"]],
+                "KRIC20": [["^_RW201DME", "A007+"], ["ITTEM", "A50+"]],
+                "KRIC34": [["^_RW341DME", "A007+"], ["SUGGR", "A50+"]]
+            },
+            "body": ["KALLI"],
+            "exitPoints": {
+                "DRAIK": ["READE", "DRAIK"],
+                "READE": ["READE"],
+                "SANNY": ["READE", "DRAIK", "SANNY"]
+            },
+            "draw": [
+                ["_RW021DME", "SUGGR", "_RW341DME"],
+                ["SUGGR", "KALLI", "READE*", "DRAIK*", "SANNY*"],
+                ["_RW161DME", "ITTEM", "_RW201DME"],
+                ["ITTEM", "KALLI"]
+            ]
+        },
+        "LUCYL6": {
+            "icao": "LUCYL6",
+            "name": "Lucyl Six",
+            "rwy": {
+                "KRIC2": [["^_RW021DME", "A007+"], ["SJAYY", "A50+"], "KAMMI"],
+                "KRIC16": [["^_RW161DME", "A007+"], ["KEGME", "A50+"], "HAYYU"],
```

Comment:
Should be KEGME at or above 4000ft, thereafter BUNKK at or above 5000ft

Reply:
Can't believe I missed an entire fix 😅 fixed